lab3/src/main/java/agh/ics/oop/Animal.java

```diff
@@ -0,0 +1,66 @@
+package agh.ics.oop;
+
+import java.util.Objects;
+
+public class Animal {
+
+    private MapDirection orientation;
+    private Vector2d position;
+
+    public Animal(){
+        this.orientation = MapDirection.NORTH;
+        this.position = new Vector2d(2,2);
+    }
+
+    public MapDirection getOrientation() {
+        return orientation;
+    }
+
+    public void setOrientation(MapDirection orientation) {
```

Comment:
setterów bym tu nie dodawał - jeśli ktoś zmieni te wartości setterami to może rozspójnić logikę poruszania się. Jedynym sposobem na modyfikowanie pozycji/orientacji powinna być metoda move(). ;)